oaknational/Oak-Web-Application
# Fix: update font size for subject principles

## Description
- Updated font sizes for subject principles on overview tab of curriculum view

## How to test

1. Go to https://deploy-preview-2069--oak-web-application.netlify.thenational.academy/teachers/curriculum/geography-primary/overview
2. Scroll down to Subject Principles
3. When viewing at a desktop size, you should see size 18px text under the title
4. When viewing at a mobile size, you should see size 16px text under the title

## Screenshots

How it used to look (delete if n/a):
<img width="1120" alt="Screenshot 2023-11-02 at 14 58 35" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/176cb431-d273-406a-9e28-b4b684dc70b9">

How it should now look:
<img width="1120" alt="Screenshot 2023-11-02 at 14 58 15" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/8ac5df2a-c220-42d2-a1f9-51a4f9e3b71f">

## Checklist
